pkg/system/phase4_configuring.go

```diff
@@ -390,6 +415,19 @@ func (r *Reconciler) SetDesiredDeploymentEndpoint() error {
 	return nil
 }
 
+func (r *Reconciler) hasNodeInclusionPolicyInPodTopologySpread() bool {
+	kubeVersion, err := util.GetKubeVersion()
+	if err != nil {
+		return true
+	}
+	for _, topologyConstraintsSkipKubeVersion := range topologyConstraintsSkipKubeVersions {
```

Comment:
you can use `semver` package to compare the versions. see usage here:
https://github.com/naveenpaul1/noobaa-operator/blob/ca4a2513254fea3f8093fa10a63d0939b37caaf8/pkg/system/phase1_verifying.go#L87-L100